src/main/frontend/src/pages/Shop.js

```diff
@@ -23,26 +23,27 @@ function Shop() {
   const { categoryId } = useParams();
   const options = [
     {
-      value: "default",
+      value: { orderBy: SORT_BY.DEFAULT, direction: ORDER.ASC },
```

Comment:
pay attention to naming here, orderBy attribute takes value from sortBy, and then direction takes value from ORDER. Why not call attributes sort and direction?